### PR TITLE
[fullstack] adds a person switching component in development

### PR DESCRIPTION
**Pull request by jzeiders**
DEV FEATURE. Inspired by stuff I used to have at work. It's adds a dropdown menu in development to let you quickly switch between active users, hella useful for role based UI's. Or if you want to setup up users with different datasets. See below for how it looks & works

Screen Shot 2020-01-27 at 8 18 15 PM

### CODE REVIEW

File: api/hackerProfile.js [hunk 1]
```diff
@@ -340,4 +340,9 @@ router.post("/undecline", async (req, res) => {
   });
 });
 
+router.get("/list", async (req, res) => {
```
Comment (WilhelmWillie) [inline api/hackerProfile.js]: iirc, this can be accessed by anyone including non-admins -- recommend adding some sort of admin check

Comment (WilhelmWillie) [reply api/hackerProfile.js]: actually to simplify this, just add a env check to see if we're in a dev environment

File: api/hackerProfile.js [hunk 2]
```diff
@@ -340,4 +340,9 @@ router.post("/undecline", async (req, res) => {
...
+  const profiles = await models.HackerProfile.findAll({});
```
Comment (saresend) [inline api/hackerProfile.js]: Nit: can just be `findAll();`

**Comment by jzeiders** [comment]
Updated with auth, sending it

**Pull request merged**